yodeploy/test_integration/helpers.py

```diff
@@ -58,15 +62,17 @@ def build_sample(app_name, version='1'):
         'PYTHONPATH': ':'.join(sys.path),
         'BUILD_NUMBER': version,
         'GIT_BRANCH': 'master',
-        'YOLA_SRC': os.path.join(tests_dir, 'samples')
+        'YOLA_SRC': os.path.join(tests_dir, 'samples'),
+        'PYTHONIOENCODING': 'utf-8'
```

Comment:
What I was getting at in the other comment is that I think it's the responsibility of the sub-process, to make sure it's got a sane output encoding. Most of the time they aren't going to be called through these `subprocess` calls that set `PYTHONIOENCODING`, so I don't think we should ever set it, to make the tests more realistic..